src/app/components/trophy/thumbnail/thumbnail.ts

```diff
@@ -34,7 +34,7 @@ export function getTrophyImg(trophy: BaseTrophy, isAchieved = true) {
 		img = 'platinum';
 	}
 
-	if (trophy.secret && !isAchieved) {
+	if (trophy.secret && !(trophy.is_achieved || trophy.has_perms)) {
```

Comment:
same comment

Reply:
can't do this here, condition is slightly different.